Settings.js

```diff
@@ -156,12 +156,13 @@ function init_settings(){
 			if (toggle.hasClass("rc-switch-checked")) {
 				toggle.click();
 			}
-			window.TOKEN_SETTINGS[setting.name] = false;
+			if (setting.name == 'hidden') toggle.click();
 		}
 		persist_token_settings(window.TOKEN_SETTINGS);
 		redraw_settings_panel_token_examples();
 	});
 	body.append(resetToDefaults);
+	resetToDefaults.click(); // Sets the default token settings
```

Comment:
This line in particular just resets all the defaults on page load.

Reply:
Yes. Sorry I misunderstood that last part and corrected my above comment. However, the token settings still don't persist in the main branch (everything is false on load).